### DISCUSSION TITLE
Not Able To Set Previous Version For Project

### DISCUSSION THREAD

**Discussion by profmitch** [coreybutler/nvm-windows#567] [General]
I am using VSCode to develop. Opened Gitbash and then PowerShell windows in admin mode and entered 'nvm use 10.13.0' and response is 'Now using node v10.13.0 (64-bit)'. Immediately enter 'node -v' and the response is 'v12.18.3'. When I do 'gulp serve' on my project (developing SharePoint web parts in Node), which is the folder containing a node_modules folder, it too says that a the previous Node version is required. What am I fundamentally doing wrong?

Also, is there a way to run Gitbash (or PowerShell) in admin mode in the VSCode integrated terminal?

**Comment by coreybutler** [comment]
Gitbash isn't technically supported, but Powershell running as an admin is. I don't really know how VS Code issues commands to the shell, nor am I familiar with a way to explicitly tell VS Code to run a shell in admin mode. You can, however, run VS Code as an admin. I suspect VS Code sends commands with the user account from which VS Code was launched, so I'd try that, using Powershell, to narrow down the problem.

**Comment by yuanoook** [comment]
I made a script to auto switch node versions by working directory for git-bash

https://gist.github.com/yuanoook/9e363ec2620d0b0e33c41e047dd25580
